### ISSUE TITLE
A problem occurred while installing modules. Even though everything is installed correctly

### ISSUE THREAD

**Issue by ItsGarek** [Pigpog/vlc-discord-rpc#56]
**Describe the bug**
It suddenly stopped working after an update.

**To Reproduce**
1. Install node
2. Unpack vlc-discord-rpc
3. Run start.bat
4. Error

**Terminal output**
A problem occurred while installing modules. Ensure that npm is installed.

**Your Setup:**
 - OS: Windows 21H2 (22000.120)
 - NodeJS version: 14.17.4

**Comment by Pigpog** [comment]
I have edited start.bat to let it show you more information, if you could download the repository using the "Code" button at the top right of [the main page](https://github.com/Pigpog/vlc-discord-rpc) and try that modified version it might provide more helpful information

**Comment by ItsGarek** [comment]
I've solved the issue. It turns out my node was somehow borked, I wiped everything and installed it from the .exe and it didn't work. So I uninstalled it, used chocolatey to install node and it started working again.

In case anyone is facing similar problems just visit: [This post on StackOverflow](https://stackoverflow.com/questions/20992723/npm-is-not-recognized-as-internal-or-external-command-operable-program-or-bat)
